presto-pinot/pom.xml

```diff
@@ -0,0 +1,464 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.227-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-pinot</artifactId>
+    <description>Presto - Pinot Native Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <dep.httpcore.version>4.4.6</dep.httpcore.version>
```

Comment:
- We usually put the dependency declaration + their version numbers in the root pom.xml.
- There are too many dependencies. I will need to dig deeper into that later on. One quick comment on this is netty has bee included by presto already. Can we reuse it?